src/components/dropdown-menu/DropdownMenu.js

```diff
@@ -242,6 +253,23 @@ class DropdownMenu extends React.Component<Props, State> {
 
         const bodyEl = bodyElement instanceof HTMLElement ? bodyElement : document.body;
 
+        const isMobile = size === VIEW_SIZE_TYPE.medium || size === VIEW_SIZE_TYPE.small;
```

Comment:
We should avoid conflating small screen size and mobile devices, as they're not mutually inclusive. Desktop users could simply have a small window, which would receive the same treatment.

Reply:
I guess isMobile is a bad name then. We wanted to have the responsive behavior for small and medium sized viewports